src/sentry/integrations/slack/command_endpoint.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from typing import List, Mapping
+
+from django.http import HttpResponse
+from rest_framework.request import Request
+
+from sentry.api.base import Endpoint
+from sentry.integrations.slack.message_builder.help import SlackHelpMessageBuilder
+from sentry.integrations.slack.requests.base import SlackRequestError
+from sentry.integrations.slack.requests.command import SlackCommandRequest
+
+logger = logging.getLogger("sentry.integrations.slack")
+
+
+def get_command(payload: Mapping[str, List[str]]) -> str:
+    text = payload.get("text", [""])[0]
```

Comment:
I think you already know this but the 0 index is grabbing the first letter instead of the word